client/App/styles/utils.scss

```diff
@@ -1,2 +1,22 @@
 $body-color: red;
 $yellow: gold;
+$second-font: "Poppins", system-ui, -apple-system, "Segoe UI", roboto, "Helvetica Neue", arial, "Noto Sans",
+  "Liberation Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
+$content-max-width: 1000px;
+$margin-auto-x: 0 auto;
+$screen-sm: 320px;
+$screen-md: 768px;
+$screen-lg: 1366px;
+$content-width-sm: 290px;
```

Comment:
add empty line between content and screen variables

Reply:
I tried but lint:fix removes this empty line